tldw_Server_API/app/main.py

```
@@ -6912,6 +6915,14 @@ async def api_metrics():
         app.include_router(sharing_router, prefix=f"{API_V1_PREFIX}", tags=["sharing"])
     except _IMPORT_EXCEPTIONS as _sharing_min_err:
         logger.debug("Skipping sharing router in minimal test app: {}", _sharing_min_err)
+    try:
+        from tldw_Server_API.app.api.v1.endpoints.prototype_workspaces import (
+            router as prototype_workspaces_router,
+        )
+
+        app.include_router(prototype_workspaces_router, prefix=f"{API_V1_PREFIX}", tags=["prototype-workspaces"])
+    except _IMPORT_EXCEPTIONS as _prototype_min_err:
+        logger.debug("Skipping prototype workspaces router in minimal test app: {}", _prototype_min_err)
```

Ruff (0.15.11)

[error] Too many arguments for `logging` format string

(PLE1205)